### PR TITLE
Add More Converters

### PR DESCRIPTION
**Pull request by eugenebokhan**

### CODE REVIEW

File: Sources/Converters.swift [hunk 1]
```diff
@@ -258,6 +347,7 @@ class ConvolutionConverter: NodeConverter {
                                             isONNX2MPS: graph.modelFormat == .mpsFlavor)
             conv = MPSCNNConvolutionTransposeNode(source: input,
                                                   weights: convDataSource)
+            conv.label = "Conv"
```
Comment (s1ddok) [inline Sources/Converters.swift]: why label transpose conv as conv?

File: Sources/Converters.swift [hunk 2]
```diff
@@ -283,20 +373,197 @@ class ConvolutionConverter: NodeConverter {
...
+            let inputShape = graph.shape(output: node.input[0])
+        else { throw ONNXGraph.Errors.noSuchOutput }
+
+        let globalAveragePool = MPSCNNPoolingAverageNode(source: input,
+                                                         kernelWidth: inputShape.width,
+                                                         kernelHeight: inputShape.height,
+                                                         strideInPixelsX: 1,
+                                                         strideInPixelsY: 1)
+        globalAveragePool.label = "GlobalAveragePool"
+        globalAveragePool.paddingPolicy = GlobalPoolPadding()
```
Comment (s1ddok) [inline Sources/Converters.swift]: I think with shape inference it can be valid only

File: Sources/Converters.swift [hunk 3]
```diff
@@ -283,20 +373,197 @@ class ConvolutionConverter: NodeConverter {
...
+            let inputShape = graph.shape(output: node.input[0])
+        else { throw ONNXGraph.Errors.noSuchOutput }
+
+        let maxPool = MPSCNNPoolingMaxNode(source: input,
+                                           kernelWidth: inputShape.width,
+                                           kernelHeight: inputShape.height,
+                                           strideInPixelsX: 1,
+                                           strideInPixelsY: 1)
+        maxPool.label = "MaxPool"
+        maxPool.paddingPolicy = GlobalPoolPadding()
```
Comment (s1ddok) [inline Sources/Converters.swift]: I don't think that's correct

File: Sources/Converters.swift [hunk 4]
```diff
@@ -16,8 +16,90 @@ enum ConvWeightArray {
     case invalid
 }
 
-// MARK: Convolutions
+// MARK: - Constant
 
+class ConstantConverter: NodeConverter {
```
Comment (RomanVolkov) [inline Sources/Converters.swift]: final

**Pull request closed without merging**